app/views/users/show/pr/_pr.html.erb

```diff
@@ -1,25 +1,90 @@
-<tr>
-  <td>
-    <div class="timeline-wrapper">
-      <div class="pr-icon">
-        <%= render partial: 'users/show/pr/icon', locals: { pr: pr } %>
+<% if valid_pr == 4 && !show_congratulations %>
```

Comment:
There seems to be a huge amount of duplicated content in this file now, shouldn't we only need the `winning-msg-cont` part inside the if?